lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -317,11 +317,11 @@ class SingleThreadQueryCountTestCase(ModuleStoreTestCase):
 
     @ddt.data(
         # old mongo with cache
-        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 12, 7),
-        (ModuleStoreEnum.Type.mongo, 50, 7, 5, 12, 7),
+        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 13, 7),
+        (ModuleStoreEnum.Type.mongo, 50, 7, 5, 13, 7),
```

Comment:
yeah, putting the staff check first is the cause - but it's coming from `can_load()` in `_has_access_descriptor` (not `_can_load_course_overview` that you commented on)